### PR TITLE
[WIP] doc: added comment for code.

### PR DESCRIPTION
**Pull request by membphis**

### CODE REVIEW

File: lua/apisix.lua [hunk 1]
```diff
@@ -43,26 +43,40 @@ function _M.http_init()
     core.id.init()
 end
 
+--[[
```
Comment (membphis) [inline lua/apisix.lua]: 这里可以删除了？好像没意义

File: lua/apisix.lua [hunk 2]
```diff
@@ -43,26 +43,40 @@ function _M.http_init()
...
 
+--]]
 function _M.http_init_worker()
+    -- 一种将事件发送到 Nginx 服务器中其他工作进程的方法。
```
Comment (membphis) [inline lua/apisix.lua]: `Nginx 服务器` -> `Nginx 服务`

File: lua/apisix.lua [hunk 3]
```diff
@@ -43,26 +43,40 @@ function _M.http_init()
...
     if not ok then
         error("failed to init worker event: " .. err)
     end
 
+    -- 加载负载均衡处理逻辑
     load_balancer = require("apisix.http.balancer").run
-
+    -- admin 处理逻辑初始化
     require("apisix.admin.init").init_worker()
+    -- 负载均衡器逻辑初始化，从配置中心加载 pstreams 配置数据
```
Comment (membphis) [inline lua/apisix.lua]: `pstreams` 打错了？

Comment (soulsoul) [reply lua/apisix.lua]: 是的 upstreams

**Comment by moonming** [comment]
@soulsoul please use English instead of Chinese for comments. thx.

**Comment by membphis** [comment]
@soulsoul Your branch is old now. How about we close this PR first? You can submit a new PR when you are ready.

**Comment by soulsoul** [comment]
ok

**Pull request closed without merging**